server/models/service-tokens.js

```diff
@@ -0,0 +1,76 @@
+const jwt = require('jsonwebtoken');
+
+// NOTE - because ServiceTokens is driven off of Sequelize ORM model
+// and not nedb (which is schemaless) I am skipping defining a Joi schema here.
+// For info on what QueryAcl schema is, see sequelize/QueryAcl.js
```

Comment:
Can you remove this comment or update?

Reply:
fixed by 6fb43c036cfe05c11a2277b95820a30bbb4a1634